tests/unit/moduleapi/reply.tcl

```diff
@@ -0,0 +1,57 @@
+set testmodule [file normalize tests/modules/reply.so]
+
+start_server {tags {"modules"}} {
+    r hello 3
```

Comment:
i think it's a good idea to repeat that test with `hello 2` too, and make sure we get the right replies.